src/scales/timeScale.ts

```diff
@@ -38,8 +38,10 @@ export module Scales {
     }
 
     protected _defaultExtent(): Date[] {
-      var endTimeValue = new Date().valueOf();
-      var startTimeValue = endTimeValue - MILLISECONDS_IN_ONE_DAY;
+      var now = new Date();
+      var endTimeValue = now.valueOf();
+      now.setDate(now.getDate() - 1);
+      var startTimeValue = now.valueOf();
       return [new Date(startTimeValue), new Date(endTimeValue)];
     }
```

Comment:
Looks like the test would probably just be the same implementation as this method, so I'm not sure it'd be helpful.